src/sql/parts/objectExplorer/common/treeNode.ts

```diff
@@ -25,6 +25,14 @@ export interface ObjectExplorerCallbacks {
 }
 
 export class TreeNode {
+	/**
+	 * USED BY NEW OE VIEWLET
```

Comment:
not sure these comments will age well...i.e., pretty soon it won't be the "new" viewlet.  Perhaps a comment that describes the purpose will be more relevant when reviewing code in the future?

Reply:
I should rename this. This is basically an artifact of trying to shim vscode extension tree apis through the OE service. I'll comment better to describe that.